ImageViewer/Source/GalleryViewController.swift

```diff
@@ -343,7 +343,17 @@ final public class GalleryViewController : UIPageViewController, UIViewControlle
     
         closeWithAnimation(programaticallyClosedCompletion)
     }
-    
+
+    public func setImageViewController(index: Int, animated: Bool) {
+
+        if currentIndex == index { return }
+        if index < 0 || index >= imageCount { return }
+        guard let imageViewController = self.imageControllerFactory.createImageViewController(index) else { return }
+
+        let direction: UIPageViewControllerNavigationDirection = index > currentIndex ? .Forward : .Reverse
+        setViewControllers([imageViewController], direction: direction, animated: animated, completion: nil)
```

Comment:
Unfortunately this needs a bit more code to make UIPageViewController happy - http://stackoverflow.com/questions/13633059/uipageviewcontroller-how-do-i-correctly-jump-to-a-specific-page-without-messing

See https://github.com/MailOnline/ImageViewer/pull/44
More specifically https://github.com/MailOnline/ImageViewer/pull/44/files#diff-c03210b1991ead2d274e1129385cad11R402